src/tiledb/cloud/soma/ingest.py

```diff
@@ -132,7 +132,7 @@ def run_ingest_workflow_udf(
             extra_tiledb_config=extra_tiledb_config,
             ingest_mode=ingest_mode,
             platform_config=platform_config,
-            resources=carry_along.get("resources", resources),
+            resources=ingest_resources,
```

Comment:
Here is where we want the UI-selected resources to land and stick.

Reply:
Please add a comment in the code, right here, saying just that. :)